Increase default results per page in UI from 10 to 50

Default value 10 items per page is too low, users hate to click on Next ->
all the time...
--- a/django-mailman3/django_mailman3/lib/paginator.py
+++ b/django-mailman3/django_mailman3/lib/paginator.py
@@ -67,7 +67,7 @@
     try:
         results_per_page = int(results_per_page)
     except (ValueError, TypeError):
-        results_per_page = 10
+        results_per_page = 50
     paginator = paginator_class(objects, results_per_page)
     try:
         objects = paginator.page(page_num)
